cnd/src/db.rs

```diff
@@ -152,6 +159,53 @@ pub struct CreatedSwap<A, B> {
     pub role: Role,
 }
 
+// This only supports han-eth/halight-btc swap.
+/// Data learned during the communication protocols that enables a created swap
+/// to be finalized.
+#[derive(Debug, Clone, Copy)]
+pub struct FinalizedSwap {
+    /// The shared swap id, generated by Bob.
+    pub swap_id: SharedSwapId,
+    /// The secret hash, generated by Alice.
+    pub secret_hash: SecretHash,
```

Comment:
I think we decided on Alice not storing the secret hash at all.

Reply:
We only discussed not storing the secret. But we could extend it to the hash for Alice. Since she computes the secret, she can also compute the hash from it. @bonomat do you have an opinion on that?

Reply:
If Alice doesn't store it then its not symmetrical for Bob and Alice, what benefit does this bring?  I'd prefer to see symmetry here.